test/Microsoft.DocAsCode.Common.Tests/FileAbstractLayerTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common.Tests
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+
+    using Xunit;
+
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Tests.Common;
+
+    [Trait("Owner", "vwxyzh")]
+    public class FileAbstractLayerTest : TestBase
+    {
+        [Fact]
+        public void TestFal_Link_Read_Simple()
```

Comment:
>TestFal_Link_Read_Simple [](start = 20, length = 24)

please follow naming convention after [http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html#unit-tests-and-functional-tests](http://dotnet.github.io/docfx/guideline/csharp_coding_standards.html#unit-tests-and-functional-tests)